yfmos.py

```diff
@@ -248,6 +248,13 @@ def __parge_B1(self, b1String):
                 pulse[i] = 'InterFrameGap'
             if self.debug:
                 print('Bucket %d: %s (%d)' % (i, pulse[i], iValue))
+	
+	if device:
+		if self.debug:
+			print('using specified device info instead of parsing B1')
+		return YfmosConfig(device=device, buckets=buckets, pulse=pulse,
+                           rollingCode=0)
+
```

Comment:
Don't mix spaces and tabs. please check your code / code-styling with `pycodestyle` before submitting a PR.

Reply:
Ok, since you are looking for another solution (split up parsing bucket and payload) I will simple close this one. 
First python steps, thanks for the input - next time.